labs/lab6/lab6.adoc

```diff
@@ -29,56 +37,311 @@ Password: INSTRUCTOR WILL PROVIDE
+
+image::images/alertmanager-red.png[]
+
+The series of actions described previously should have been executed:
+
+1. Prometheus notifies AlertManager
+2. AlertManager parses the alert, passes it to the receiver and invokes the AlertManager Tower Bridge application
+3. The AlertManager Tower Bridge Application parses the alert and invokes the Workflow Job in Ansible Tower
+4. The Ansible Tower Workflow job runs to completion.
+
+Let's validate that this occurred properly.
+
+First, navigate to the AlertManager console or the tab that was previously open.
+
+link:https://alertmanager-openshift-metrics.apps-<student_id>.labs.sysdeseng.com[https://alertmanager-openshift-metrics.apps-<student_id>.labs.sysdeseng.com]
+
+An alert should now be indicated on the _Alerts_ page as shown below.
+
+image::images/alertmanager-console.png[]
+
+Notice how the `type=ansible-tower` is prominently displayed.
+
+Now, verify the AlertManager Tower Bridge application has received the webhook and invoked Ansible Tower.
+
+Navigate to the OpenShift web console and login using the _<student_id>-admin_ account using the provided credentials.
+
+link:https://master-<student_id>.labs.sysdeseng.com:8443[https://master-<student_id>.labs.sysdeseng.com:8443]
+
+On the righthand side of the page, select the _openshift-metrics_ project. 
+
+From within the _openshift-metrics_ project, select **Applications** and then **Pods** from the lefthand side navigation. 
+
+Locate the _Running_ pod starting with `alertmanager-tower-bridge` and then select the **Logs** tab. When a request is received, details of the alert along with the invocations to Ansible Tower are displayed. The final invocation is the actual request to launch the workflow job template. The response will include an the `id` of the job that was triggered. The presence of this field indicates the invocation was successful and can be used to correlate an action within Ansible Tower.
+
+The final step to verify the status of the **2-Scaleup_OpenShift_on_AWS** workflow job template was invoked expand the OpenShift cluster.
+
+In the web browser, navigate to to Ansible Tower and login if an active session has not been retained
+
+link:https://tower-<student_id>.labs.sysdeseng.com[https://tower-<student_id>.labs.sysdeseng.com] 
+
+Select the **JOBS** link on the navigation bar. Locate and select the ID of the job retrieved previously from the AlertManager Tower Bridge Application. The progress of the workflow job is then displayed
+
+image::images/ansible-tower-scaleup-workflow.png[]
 
 The job will take a few minutes to complete. Monitor the status until the workflow job completes successfully by selecting **Details** on each job as with the initial workflow job used to install the cluster initially.
```

Comment:
It actually takes a long time to finish.  Maybe don't have them wait?  Not sure what to do here.